core/unit_test/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ TARGET_COMPILE_DEFINITIONS(kokkos_gtest PUBLIC GTEST_HAS_TR1_TUPLE=0 GTEST_HAS_P
 
 TARGET_INCLUDE_DIRECTORIES(kokkos_gtest PUBLIC ${GTEST_SOURCE_DIR})
 #Gtest minimally requires C++11
-IF(NOT (Kokkos_ENABLE_CUDA AND WIN32))
+IF((NOT (Kokkos_ENABLE_CUDA AND WIN32)) AND (NOT ("${KOKKOS_CXX_COMPILER_ID}" STREQUAL "Fujitsu")))
 TARGET_COMPILE_FEATURES(kokkos_gtest PUBLIC cxx_std_11)
```

Comment:
Would someone remind me why we need this in the first place?  Don't we set `CMAKE_CXX_STANDARD`?

Reply:
I thought someone found this was needed on some platform ...

Reply:
AFAIK, we don't compile `gtest` with the `Kokkos` flags and we don't require `CMAKE_CXX_STANDARD` directly. Hence, there were some issues in the nightlies at some point.

Reply:
I would be surprised if we still need that line at all but this probably shouldn't be addressed here

Reply:
I was suprised we had issues to begin with. And yes somewhere we needed that with some combination of weird compiler (Intel XL or so) and some cmake version.